nav_msgs/Odometry chassis_odom
nav_msgs/Odometry[] arm_odom
int16 gripper_state